wolfcrypt/src/rsa.c

```diff
@@ -2677,8 +2677,7 @@ static int RsaPrivateDecryptEx(byte* in, word32 inLen, byte* out,
         ret = wc_RsaFunction(key->data, inLen, key->data, &key->dataLen,
                                                             rsa_type, key, rng);
 #else
-        ret = wc_RsaFunction(out, inLen, out, &key->dataLen, rsa_type, key,
-                                                                           rng);
+        ret = wc_RsaFunction(in, inLen, out, &key->dataLen, rsa_type, key, rng);
```

Comment:
This is the INLINE case but you've changed the parameters to NON INLINE.  That seems wrong.

Reply:
It is the INLINE case that I changed. Did I miss something?

Reply:
I guess the parameters are still inline when passed inline because in will equal out.  But when not passed inline they need to be the way you have them.  Thanks.